From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Andrew Kaster <akaster@serenityos.org>
Date: Sun, 23 Jan 2022 17:55:32 -0700
Subject: [PATCH] Disable mimalloc for serenity

mimalloc needs some help to compile and run on serenity.
That's one yak too far for right now.
---
 Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Makefile b/Makefile
index 946c952..dc8563d 100644
--- a/Makefile
+++ b/Makefile
@@ -60,6 +60,8 @@ endif
 USE_MIMALLOC = 1
 ifeq ($(OS), Darwin)
   USE_MIMALLOC = 0
+else ifeq ($(OS), SerenityOS)
+  USE_MIMALLOC = 0
 else ifeq ($(IS_ANDROID), 1)
   USE_MIMALLOC = 0
 endif
